code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -10,3 +10,117 @@
+			item_state = ""
+			force = WEAPON_FORCE_WEAK
+			throwforce = WEAPON_FORCE_WEAK
+			origin_tech = list(TECH_COMBAT = 1)
+
+
+/obj/item/weapon/melee/toolbox_maul/update_icon()
+	..()
+	overlays.Cut()
+	if(reinforced)
+		overlays += "[icon_state]-duct_tape"
+
+/obj/item/weapon/melee/toolbox_maul/proc/breaking(var/mob/living/user)
+	qdel(src)
+	var/obj/item/weapon/mop/mop = new(user.loc)
+	if(!user.get_active_hand())
+		user.put_in_active_hand(mop)
+	else
+		user.put_in_inactive_hand(mop)
+	toolbox.loc = user.loc
+
+
+/obj/item/weapon/melee/toolbox_maul/attackby(obj/item/C, mob/living/user)
+	if(toolbox)
+		if(istype(C, /obj/item/weapon/tool/wirecutters))
+			if(reinforced)
+				user << SPAN_NOTICE("You cutted up the tapes from [src].")
+				reinforced = FALSE
+			else
+				user << SPAN_NOTICE("You carefully cut cables from [src].")
+				breaking(user)
+
+		if(istype(C, /obj/item/weapon/tape_roll))
+			user << SPAN_NOTICE("You begins to tie [src] with [C]...")
+			if(do_after(user, 50))
+				if(!reinforced)
+					user << SPAN_NOTICE("You reinforce [src].")
+					reinforced = TRUE
+				else
+					user << SPAN_WARNING("[src] is already reinforced.")
+	else
+		if(istype(C, /obj/item/weapon/storage/toolbox))
+			src.name = initial(src.name)
+			src.desc = initial(src.desc)
+			src.force = initial(src.force)
+			throwforce = initial(throwforce)
+			origin_tech = initial(origin_tech)
+			icon_state = initial(icon_state)
+			item_state = initial(item_state)
+			toolbox = C
+			user.drop_from_inventory(C, src)
+			if(istype(C, /obj/item/weapon/storage/toolbox/electrical))
+				icon_state = "hm_hammer_yellow"
+				item_state = "hm_hammer_yellow"
+			if(istype(C, /obj/item/weapon/storage/toolbox/mechanical))
+				icon_state = "hm_hammer_blue"
+				item_state = "hm_hammer_blue"
+			user << SPAN_NOTICE("You tied [C] to [src] and finally finish it!")
+	update_icon()
+
+/obj/item/weapon/melee/toolbox_maul/attack(mob/living/carbon/human/M as mob, mob/living/carbon/user as mob)
+	..()
+	if(!reinforced && prob(5))
+		breaking(user)
+		playsound(src.loc, 'sound/effects/bang.ogg', 45, 1)
+		user.visible_message(SPAN_WARNING("[src] breaks in hands of [user]!"))
+
+
+/obj/item/weapon/melee/nailstick
+	name = "nailed stick"
+	desc = "Stick with some nails in it. Looks sharp enough."
+	icon_state = "hm_spikeclub"
+	item_state = "hm_spikeclub"
+	force = WEAPON_FORCE_NORMAL
+	throwforce = WEAPON_FORCE_NORMAL
+	w_class = ITEM_SIZE_NORMAL
+	origin_tech = list(TECH_COMBAT = 2)
+	attack_verb = list("beaten", "slammed", "smacked", "struck", "battered")
+
+/obj/item/weapon/melee/nailstick/proc/attack_type_change()
+	if(sharp)
+		sharp = FALSE
+		attack_verb = list("beaten", "slammed", "smacked", "struck", "battered")
+		force = WEAPON_FORCE_WEAK
+	else
+		sharp = TRUE
+		attack_verb = list("scratched", "sliced", "slashed")
+		force = WEAPON_FORCE_NORMAL
+
+/obj/item/weapon/melee/nailstick/attack(mob/living/carbon/human/M as mob, mob/living/carbon/user as mob)
+	if(prob(30))
```

Comment:
What's the purpose of this exactly? IT has a 30% chance to switch back and forth between attack types? Why?

If its supposed to represent the nails breaking, you should do something to make it stick
Also the Sharp property never seems to get changed